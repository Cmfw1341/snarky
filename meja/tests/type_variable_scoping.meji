let f : int -> int;  let g : bool -> bool; 
module X :
  {
    type t('a) = A('a) | B(int) | C(bool) : X.t(bool);
    
    let a : X.t(int);
    
    let a1 : X.t(unit);
    
    let b : X.t('bool);
    
    let c : X.t(bool);
    
    let f : 'int -> 'int -> 'int;
    
    }; 

