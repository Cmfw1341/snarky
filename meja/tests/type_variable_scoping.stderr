File "tests/type_variable_scoping.meja", line 18, characters 23-27:
Error: Unbound constructor true.
